tests/test_helpers.py

```
@@ -43,3 +46,68 @@ def test_coerce_bool_parses_bool_like_values(value: Any, expected: bool) -> None
+        return MagicMock()
+
+    class _CookieJar:
+        def __init__(self, *, unsafe: bool) -> None:
+            """Capture the unsafe flag without requiring a running event loop."""
+            self._unsafe = unsafe
+
+    monkeypatch.setattr(helpers_mod, "async_create_clientsession", _async_create_clientsession)
+    monkeypatch.setattr(helpers_mod.aiohttp, "CookieJar", _CookieJar)
+    monkeypatch.setattr(helpers_mod, "OPNsenseClient", _client)
+
+    password = "pass"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "password"

(S105)